script/seed.js

```diff
@@ -12,7 +12,25 @@ async function seed() {
     User.create({email: 'murphy@email.com', password: '123'})
   ])
 
+  const products = await Promise.all([
+    Product.create({
+      title: 'Shepards Pie',
+      description:
+        'Give your cat a trip to the English country side with minced red meat, cooked in a gravy with onions, vegetables, and topped with catnip',
+      price: 2.99,
```

Comment:
Hot tip: don't store money as floats. Store in terms of cents (integers), display in terms of dollars and cents (divide by 100). Integer math is accurate, floating point math is not.

Reply:
Follow-up, because "price" is then ambiguous, you may want something like `price_cents` to make it 100% clear what's going on.

Reply:
Thanks, Gabe. I take your point.

Do you mean, then, something like:
```

  priceCents: {
    type: Sequelize.INTEGER,
    allowNull: false,
    validate: {
      min: 0
    }
  }
```
...Or would that not be the whole story? I'm thinking we could handle display somewhere on the front-end, if this syntax doesn't cause calculation issues.

Reply:
Yes that's what I meant (more or less, I wasn't thinking deeply about implementation, just high-level strategy).

Reply:
I understand. Thanks for this and all the other tips. I've made revisions accordingly.

Reply:
Sure thing.

On a fun side note, in a typed language like Haskell I'd invent a new numerical type like this:

```hs
{-# GeneralizedNewtypeDeriving #-}

newtype Cents = Cents Int deriving Num
```

Then my functions / variables could declare that they _must_ be cents:

```hs
addTwoBucks :: Cents -> Cents
addTwoBucks c = c + 200
```

If I tried to pass in a similar "dollars" amount, the compiler would yell at me:

```hs
myWallet = 500 :: Cents
giftFromGrandpa = 10 :: Dollars
myNewWallet = myWallet + giftFromGrandpa -- ERROR, does not compile
```

In this way you can make it impossible for someone to accidentally add an amount stored as dollars to an amount stored as cents.

In JS, you just have to be very very careful… lest your [$125M space probe goes down](http://www.cnn.com/TECH/space/9909/30/mars.metric.02/).

Reply:
Lol that article. This is the kind of thing that makes me more curious about TypeScript, if I understand what people have been telling me about what it does. I've gotta read about that, finally...

Reply:
PPS Actually in Haskell I would use the `safe-money` library. 
https://ren.zone/articles/safe-money